image.yaml

```diff
@@ -59,7 +59,7 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_725_CR2
```

Comment:
@drichtarik Maybe the cct_module above would be bumpable to [`0.39.0`](https://github.com/jboss-openshift/cct_module/releases/tag/0.39.0) version too?

Reply:
OK. I didn't notice the tag since I checked the releases tab only.

Reply:
Thanks